Marlin/language_en.h

```diff
@@ -508,5 +508,8 @@
 #ifndef MSG_DELTA_CALIBRATE_CENTER
   #define MSG_DELTA_CALIBRATE_CENTER          "Calibrate Center"
 #endif
-
+#define MSG_CHANGE_FILAMENT                 "Filament"
+#define MSG_REMOVE_FILAMENT                 "Remove"
+#define MSG_INSERT_FILAMENT                 "Insert"
+#define MSG_INSERT_FILAMENT_SLOW            "Slow Insert"
 #endif // LANGUAGE_EN_H
```

Comment:
> Why do we have that strange #ifndef #endif structures around the definitions in the English language file?

The English language file is always included. It ensures that all strings are defined, including those which may be missing in the selected language.